README.md

```diff
@@ -1,20 +1,32 @@
-[![Documentation Status](https://readthedocs.org/projects/ai4s2s/badge/?version=latest)](https://ai4s2s.readthedocs.io/en/latest/?badge=latest)
 [![github license badge](https://img.shields.io/github/license/AI4S2S/s2spy)](https://github.com/AI4S2S/s2spy)
-[![fair-software badge](https://img.shields.io/badge/fair--software.eu-%E2%97%8F%20%20%E2%97%8F%20%20%E2%97%8F%20%20%E2%97%8F%20%20%E2%97%8B-yellow)](https://fair-software.eu) \
+[![fair-software badge](https://img.shields.io/badge/fair--software.eu-%E2%97%8F%20%20%E2%97%8F%20%20%E2%97%8F%20%20%E2%97%8F%20%20%E2%97%8B-yellow)](https://fair-software.eu)
 [![build](https://github.com/AI4S2S/s2spy/actions/workflows/build.yml/badge.svg)](https://github.com/AI4S2S/s2spy/actions/workflows/build.yml)
 [![sonarcloud](https://github.com/AI4S2S/s2spy/actions/workflows/sonarcloud.yml/badge.svg)](https://github.com/AI4S2S/s2spy/actions/workflows/sonarcloud.yml)
 [![workflow scc badge](https://sonarcloud.io/api/project_badges/measure?project=AI4S2S_ai4s2s&metric=coverage)](https://sonarcloud.io/dashboard?id=AI4S2S_ai4s2s)
-<!-- [![RSD](https://img.shields.io/badge/rsd-s2s-00a3e3.svg)](https://www.research-software.nl/software/s2s) -->
-<!-- [![DOI](https://zenodo.org/badge/DOI/<replace-with-created-DOI>.svg)](https://doi.org/<replace-with-created-DOI>) -->
+
+A high-level python package integrating expert knowledge and artificial intelligence to boost (sub) seasonal forecasting.
+
+## Why s2spy?
+Producing reliable (sub) seasonal (S2S) forecasts with machine learning techniques remains a huge challenge. Currently, these data-driven S2S forecasts generally suffer from a lack of trust because:
+- Intransparent data processing and poorly reproducible scientific outcomes
+- Technical pitfalls related to machine learning-based predictability (e.g. overfitting)
+- Black-box methods without sufficient explanation
+
+To tackle these challenges, we build this open-source, high-level python package, `s2spy`. It provides an interface between artificial intelligence and expert knowledge, to boost predictability and physical understanding of S2S processes. By implementing optimal data-handling and parallel-computing packages, it can efficiently run across different Big Climate Data platforms. Key components will be explainable AI and causal discovery, which will support the classical scientific interplay between theory, hypothesis-generation and data-driven hypothesis-testing, enabling knowledge-mining from data.
+
+This tool will be a community effort. It can help us achieve trustworthy data-driven forecasts by providing the user with:
+- Transparent and reproducible analyses
+- Best practices in model verification
+- Understanding the sources of predictability
 
 ## Installation
+[![workflow pypi badge](https://img.shields.io/pypi/v/s2spy.svg?colorB=blue)](https://pypi.python.org/project/s2spy/)
+[![supported python versions](https://img.shields.io/pypi/pyversions/s2spy)](https://pypi.python.org/project/s2spy/)
```

Comment:
the python versions here ['3.7', '3.8', '3.9'] are not the same as those used in the [build workflow ](https://github.com/AI4S2S/s2spy/blob/8c8251e20965195c8f90843292e846ac30ce5cb8/.github/workflows/build.yml#L20) i.e. ['3.8', '3.9', '3.10']

Reply:
Thanks for the notice. Yes, this is because in our last release the python versions are ['3.7', '3.8', '3.9'] and pypi has these versions. But now we switch to ['3.8', '3.9', '3.10']. Since we haven't made a release, on pypi it is still ['3.7', '3.8', '3.9']. The badges are loaded from pypi. This will be updated after we make a new release. But thanks for mentioning this. 👍